Unreal/CMakeLists.txt

```diff
@@ -337,9 +340,11 @@ add_dependencies (
   ${UE_DEPENDENCIES_ORDER_ONLY}
 )
 
-function(add_carla_ue_package_target TARGET_NAME_SUFFIX UE_BUILD_CONFIGURATION)
-  add_custom_target (
+function (add_carla_ue_package_target PACKAGE_CONFIGURATION UE_BUILD_CONFIGURATION)
+  set (TARGET_NAME_SUFFIX -${PACKAGE_CONFIGURATION})
```

Comment:
In the case that `PACKAGE_CONFIGURATION` is empty, target suffix will be `-` and this will generate the target `pacakge-` instead `package`.

We should generate and empty suffix when `PACKAGE_CONFIGURATION` is empty to maintain the target name `package`.